<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioTestWindow</class>
 <widget class="QMainWindow" name="AudioTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AudioTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="generateCheckBox">
      <property name="text">
       <string>Generate sound</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="generatorGroupBox">
      <property name="title">
       <string>Generator:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="readCheckBox">
      <property name="text">
       <string>Read sound</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="readerGroupBox">
      <property name="title">
       <string>Reader:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="inputLevelLabel">
         <property name="text">
          <string>Input level : -</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>523</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuControls"/>
  </widget>
  <action name="actionSet_TC_In">
   <property name="text">
    <string>Set TC In...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionSet_TC_Out">
   <property name="text">
    <string>Set TC Out...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
